reports/2019-05-31.md

```diff
@@ -0,0 +1,114 @@
+# Report for 2019-05-01 to 2019-05-31
+
+This report covers the development in the [linuxkit/linuxkit] [linuxkit/lcow] [linuxkit/kubernetes] [linuxkit/virtsock] [linuxkit/rtf] [linuxkit/homebrew-linuxkit] [moby/tool] repositories. There were 78 contributions (PRs/Issues/Comments) from 24 individual contributors. 15 new PRs were opened and 8 PRs were merged. 10 new issues were opened and 7 issues were closed.
+
+Along with the usual updates to the kernel and WireGuard, this months has seen a bunch of changes to support the Raspberry Pi Model 3b+.
+
+## Merged PRs:
+
+- Fix issue wherein multiple files named kernel in the tar break various mkimage utils ([linuxkit/linuxkit#3341] [@deitch] [@rn] [@ijc])
+- Update kernels to 5.0.12/4.9.39/4.14.116/4.9.173 ([linuxkit/linuxkit#3344] [@rn] [@justincormack])
+- Update kernels to 5.0.15/4.19.42/4.14.118/4.9.175 and add 5.1.1 kernel ([linuxkit/linuxkit#3347] [@rn] [@justincormack])
+- April 2019 report and authors update ([linuxkit/linuxkit#3348] [@rn] [@ijc])
+- kernel: Include lan78xx kernel module for use with rpi3b+ ([linuxkit/linuxkit#3350] [@schrotthaufen] [@rn])
+- Update referenced Raspberry Pi firmware ([linuxkit/linuxkit#3351] [@riconnon] [@rn])
+- Update to WireGuard  0.0.20190406 ([linuxkit/linuxkit#3355] [@rn] [@justincormack])
+- Upte kernels to 5.1.5/5.0.19/4.19.46/4.14.122/4.9.179 ([linuxkit/linuxkit#3358] [@rn] [@justincormack])
```

Comment:
typo

Reply:
it was a typo in the PR summary. My script just extracts them as is